CHANGES.rst

```diff
@@ -2,7 +2,7 @@
 Changelog
 =========
 
-plateau 4.0.4 (2022-02-XX)
+Plateau 4.1.0 (2022-03-16)
```

Comment:
This should be 4.0.4, the release only contains bug fixes.

Reply:
Done.